ClinGen/clincoded
# Connects to #1706. Dashboard loading speed of GDMs and Interpretations.

**Prerequisite:**
The testing steps below require production data to be loaded in the test instance.

**Steps to test:**
1. Assign yourself to affiliation **10007** upon logging into the interface with your **stanford.edu** email account.
2. Logout and login again.
3. Upon seeing the shown affiliation modal, choose the **Hearing Loss EP** affiliation.
4. Before clicking the **Continue** button in the modal, open the **Chrome Developer Tools**.
5. In the **Chrome Developer Tools**, select the **Network** tab: 
![image](https://user-images.githubusercontent.com/6956310/41570863-613e400c-7325-11e8-9888-5f3063bb7d36.png)
6. Then select the **XHR** subtab:
![image](https://user-images.githubusercontent.com/6956310/41570894-884fdcb4-7325-11e8-8fba-04c2502ee934.png)
7. Click on the **Clear** icon to remove all the existing logged requests:
![image](https://user-images.githubusercontent.com/6956310/41570929-b09c29e8-7325-11e8-8638-fb527a8412d7.png)
6. Now click the **Continue** button in the modal.
7. In the **Network** subtab of the **Chrome Developer Tools**, expect to see a list of requests being logged.
![image](https://user-images.githubusercontent.com/6956310/41751652-5a482a02-7577-11e8-8cee-1a5305a673cd.png)
8. Mouse over the _last_ yellow bar in the **Waterfall** column, expect to see the total time for completing this request to be less than 5 seconds given the number of GDMs and Interpretations currently associated with this affiliation.
